components/datetime/src/neo_marker.rs

```diff
@@ -1558,7 +1558,7 @@ macro_rules! impl_zoneddatetime_marker {
 impl_day_marker!(
     NeoYearMonthDayMarker,
     NeoDayComponents::YearMonthDay,
-    description = "a Year/Month/Day format",
+    description = "year, month, and day (era elided when possible)",
```

Comment:
`NeoNeverMarker` says it's "A struct implementing traits for never loading data.", however these markers now just say "“3:47:50 PM” ⇒ locale-dependent time fields". They should say "A struct implementing traits for loading locale-depdendent time fields (e.g. "3:47:50 PM")."

Maybe the examples are better as a table in this module, because there are `FullDataCalMarkers`, `NeoNeverMarker`, and `NoDataCalMarkers` floating around here, which are markers but not semantic skeleta.

Reply:
Oh I definitely plan to move these markers around. I was just cleaning up the docs in the mean time. This isn't the final home of these marker types. I'm asking for a review of the markers implementing DateTimeMarkers, not a review of the other markers implementing other scaffolding traits.